trackintel/preprocessing/triplegs.py

```diff
@@ -5,89 +5,29 @@
 import pandas as pd
 from tqdm import tqdm
 
-def smoothen_triplegs(triplegs, method='douglas-peucker', tolerance=1.0):
-    """reduces number of points while retaining structure of tripleg
+def smoothen_triplegs(triplegs, tolerance=1.0):
+    """Reduces number of points while retaining structure of tripleg
```

Comment:
It would be nice to add the used method in the comments or to at least refer to the function that we wrap
(I am aware that you cleaned the code and that it was worse before ;-) )

Reply:
A simple description is added